client/src/js/services/DocumentService.js

```diff
@@ -0,0 +1,40 @@
+angular.module('bhima.services')
+.service('DocumentService', DocumentService);
+
+DocumentService.$inject = [ '$http', 'util' ];
+
+function DocumentService($http, util) {
+  var service = this;
+  var baseUrl = '/patients/';
+
+  /** expose the service */
+  service.read = read;
+  service.remove = remove;
+  service.removeAll = removeAll;
+
+  /**
+   * This method returns documents on a patient given the patients UUID.
+   *
+   * @param {String|Null} uuid   The patient's UUID  (could be null)
+   * @return {Object}       Promise object that will return patient details
+   */
+  function read(patientUuid) {
+    if (!patientUuid) { return ; }
+
+    return $http.get(baseUrl.concat(patientUuid).concat('/documents'))
+      .then(util.unwrapHttpResponse);
+  }
+
+  /** delete document */
+  function remove(patientUuid, documentUuid) {
+    return $http.delete(baseUrl.concat(patientUuid).concat('/documents/').concat(documentUuid))
+      .then(util.unwrapHttpResponse);
+  }
+
+  /** delete all document */
+  function removeAll(patientUuid, documentUuid) {
```

Comment:
The actual API implementation does not have this route.  To delete all documents, the implementations looks like: `DELETE patients/:uuid/documents/all`.   See [here](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/server/controllers/medical/patients/documents.js#L111) for reference and [here](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/server/config/routes.js#L358) for the binding.